GAE/war/WEB-INF/rest-servlet.xml

```diff
@@ -37,6 +37,7 @@
     <bean name="/question_groups" class="org.waterforpeople.mapping.app.web.rest.QuestionGroupRestService" lazy-init="true" />
     <bean name="/question_options" class="org.waterforpeople.mapping.app.web.rest.QuestionOptionRestService" lazy-init="true" />
     <bean name="/questions" class="org.waterforpeople.mapping.app.web.rest.QuestionRestService" lazy-init="true" />
+    <bean name="/reports" class="org.akvo.flow.rest.ReportRestService" />
```

Comment:
make it a `lazy-init` class as well so that it only initialised by spring when we are actually going to use it.

Reply:
ok